src/sentry/mediators/sentry_app_installations/destroyer.py

```diff
@@ -41,6 +46,21 @@ def _destroy_service_hooks(self):
         for hook in hooks:
             service_hooks.Destroyer.run(service_hook=hook)
 
+    def _destroy_api_tokens(self):
+        tokens = ApiToken.objects.filter(
+            id__in=SentryAppInstallationToken.object.filter(
+                sentry_app_installation_id=self.install.id,
+            ).values_list('api_token_id', flat=True),
+        )
+
+        for token in tokens:
+            sentry_app_installation_tokens.Destroyer.run(
```

Comment:
@MeredithAnya Finding the API tokens associated with a set of `SentryAppInstallationToken`, then passing in both the install and the api token as the arguments to the destroyer seems a bit convoluted to me. Why not just get all the `SentryAppInstallationToken` objects associated with the installation then passing only the `SentryAppInstallationToken` into the destroyer? The destroyer can query for the associated objects to destroy.

Reply:
I think this kind of depends on how the `DELETE` endpoint will look. Were you thinking of passing in the `SentryAppInstallationToken` in there?